### PR TITLE
FE-253: bugfix - validate JSON, show errors & disable save when error

### PR DESCRIPTION
**Pull request by palashkaria**
### Scope & Purpose

On saving invalid JSON, we were getting a 'success' message; this fixes this by doing client side JSON validation. Shows error when JSON is invalid and disables the save button
- [x] :hankey: Bugfix

### Checklist

- [x] Tests
  - [x] Manually tested
- [x] :book: CHANGELOG entry made
- [x] Backports
  - [x] Backport for 3.11: https://github.com/arangodb/arangodb/pull/18804

#### Related Information

- [x] GitHub issue / Jira ticket: https://arangodb.atlassian.net/browse/FE-253

### CODE REVIEW

File: js/apps/system/_admin/aardvark/APP/react/src/components/jsonEditor/JSONErrors.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      paddingX="2"
+      paddingY="1"
+      fontSize="sm"
+      color="red"
+      background="red.100"
+      overflow={"auto"}
+    >
+      {errors.map(error => {
+        return (
+          <Box>{`${error.keyword} error: ${
```
Comment (nikhil-varma) [inline js/apps/system/_admin/aardvark/APP/react/src/components/jsonEditor/JSONErrors.tsx:21]: It's not asking for a `key` prop in console? 🤔

**Review by nikhil-varma — COMMENTED** [review]
Minor question however,

**Review by nikhil-varma — APPROVED** [review]
Approving it considering the comment is addressed.

File: js/apps/system/_admin/aardvark/APP/react/src/components/jsonEditor/JSONErrors.tsx (hunk 1 above)
Comment (palashkaria) [reply js/apps/system/_admin/aardvark/APP/react/src/components/jsonEditor/JSONErrors.tsx:21]: Oh yes, this component was already there and I missed it. Thanks!

**Pull request merged**